collections/admin/importextended.php

```diff
@@ -337,11 +338,14 @@ function importTypeChanged(selectElement){
 									<option value="externalOccurrence"><?= $LANG['EXTERNAL_OCCURRENCE'] ?></option>
 									<option value="observational"><?= $LANG['OBSERVATION'] ?></option>
 								</select>
+								<div class="top-breathing-room-rel danger" style="color: var(--danger-color);">
+									<caption><?= $LANG['ASSOCIATION_UPLOAD_WARNING'] ?></caption>
```

Comment:
A little behavioral nudge for the end users who could footgun themselves with bulk uploading associations matching to the correct catalogNumber in an off-target collection (since cross-collection associations are correctly allowed as well).